rosbag2_py/CMakeLists.txt

```diff
@@ -26,46 +26,17 @@ find_package(rosbag2_transport REQUIRED)
 # Find python before pybind11
 find_package(python_cmake_module REQUIRED)
 find_package(PythonExtra REQUIRED)
+find_package(Python3 REQUIRED Interpreter Development)
 if(WIN32 AND CMAKE_BUILD_TYPE STREQUAL "Debug")
   # Set the python debug interpreter.
-  # pybind11 will setup the build for debug now.
+  # We need to set *both* PYTHON_EXECUTABLE (for use in this CMake file),
```

Comment:
Same comment as on the rclpy PR about doing something like ros2/geometry2#494

Reply:
Cleaned up in 6919813